{"uid":"d9c2b9eaa9e1af6b","name":"test_build_and_deploy","fullName":"tests.integration.metrics_tests.test_metrics#test_build_and_deploy","historyId":"122a2ca65473b510dab9ec05ca155281","time":{"start":1724463614602,"stop":1724464245545,"duration":630943},"description":"Build and deploy one unit of MongoDB.","descriptionHtml":"<p>Build and deploy one unit of MongoDB.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmongodb/1 [executing] maintenance: installing MongoDB","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/tests/integration/metrics_tests/test_metrics.py\", line 43, in test_build_and_deploy\n    await ops_test.model.wait_for_idle()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2956, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\nmongodb/1 [executing] maintenance: installing MongoDB","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1724463614428,"stop":1724463614429,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1724463614429,"stop":1724463614429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1724463614429,"stop":1724463614601,"duration":172},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1724463614601,"stop":1724463614601,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy one unit of MongoDB.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmongodb/1 [executing] maintenance: installing MongoDB","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/tests/integration/metrics_tests/test_metrics.py\", line 43, in test_build_and_deploy\n    await ops_test.model.wait_for_idle()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2956, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\nmongodb/1 [executing] maintenance: installing MongoDB","steps":[],"attachments":[{"uid":"855d277881454779","name":"log","source":"855d277881454779.txt","type":"text/plain","size":3433}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1724464245551,"stop":1724464245551,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1724464245561,"stop":1724464246009,"duration":448},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1724464246009,"stop":1724464246009,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1724464246009,"stop":1724464246010,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1724464246010,"stop":1724464246010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='skipping deploy, model expected to be provided.')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.metrics_tests"},{"name":"suite","value":"test_metrics"},{"name":"host","value":"openstack-amd64-large-ps5-2-e526a4b9a89d3587cddf13a2"},{"name":"thread","value":"10641-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.metrics_tests.test_metrics"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"5729cf165c345820","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":23,"unknown":1,"total":26},"items":[{"uid":"c65e78498a5de779","reportUrl":"../2173//#testresult/c65e78498a5de779","status":"passed","time":{"start":1724378525974,"stop":1724378771883,"duration":245909}},{"uid":"16bf127b94d58b88","reportUrl":"../2163//#testresult/16bf127b94d58b88","status":"unknown","time":{}},{"uid":"a04f51882f0912de","reportUrl":"../2152//#testresult/a04f51882f0912de","status":"passed","time":{"start":1724204118274,"stop":1724204266148,"duration":147874}},{"uid":"ca37f8ee015db4a6","reportUrl":"../2126//#testresult/ca37f8ee015db4a6","status":"passed","time":{"start":1724117735146,"stop":1724117970047,"duration":234901}},{"uid":"a33535ed19549288","reportUrl":"../2113//#testresult/a33535ed19549288","status":"passed","time":{"start":1724031405871,"stop":1724031731920,"duration":326049}},{"uid":"9afba5d3ebfc0f7e","reportUrl":"../2112//#testresult/9afba5d3ebfc0f7e","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmongodb/1 [executing] maintenance: installing MongoDB","time":{"start":1723945242733,"stop":1723945854332,"duration":611599}},{"uid":"b4f53963e41eb1d5","reportUrl":"../2111//#testresult/b4f53963e41eb1d5","status":"passed","time":{"start":1723858483631,"stop":1723858705847,"duration":222216}},{"uid":"3902d0563d6485fc","reportUrl":"../2102//#testresult/3902d0563d6485fc","status":"passed","time":{"start":1723772242329,"stop":1723772495624,"duration":253295}},{"uid":"d6333e6d0166354a","reportUrl":"../2101//#testresult/d6333e6d0166354a","status":"passed","time":{"start":1723685813614,"stop":1723686121780,"duration":308166}},{"uid":"1da8ccac5fb92d7e","reportUrl":"../2096//#testresult/1da8ccac5fb92d7e","status":"passed","time":{"start":1723599898813,"stop":1723600191120,"duration":292307}},{"uid":"5593b4c29d6173bc","reportUrl":"../2090//#testresult/5593b4c29d6173bc","status":"passed","time":{"start":1723514120652,"stop":1723514457898,"duration":337246}},{"uid":"82f831a25fa4fd48","reportUrl":"../2087//#testresult/82f831a25fa4fd48","status":"passed","time":{"start":1723426867459,"stop":1723427164525,"duration":297066}},{"uid":"271e055f39be4053","reportUrl":"../2086//#testresult/271e055f39be4053","status":"passed","time":{"start":1723340492465,"stop":1723340845755,"duration":353290}},{"uid":"b9848cb40e326f7","reportUrl":"../2085//#testresult/b9848cb40e326f7","status":"passed","time":{"start":1723253881030,"stop":1723254174990,"duration":293960}},{"uid":"d66aa1bd4fae9529","reportUrl":"../2076//#testresult/d66aa1bd4fae9529","status":"passed","time":{"start":1723168037536,"stop":1723168365058,"duration":327522}},{"uid":"defb69ab71731bc","reportUrl":"../2073//#testresult/defb69ab71731bc","status":"passed","time":{"start":1723084086355,"stop":1723084403106,"duration":316751}},{"uid":"74a97e68f5c72cf2","reportUrl":"../2067//#testresult/74a97e68f5c72cf2","status":"passed","time":{"start":1722995139275,"stop":1722995452252,"duration":312977}},{"uid":"46e64dddf1b0301d","reportUrl":"../2056//#testresult/46e64dddf1b0301d","status":"passed","time":{"start":1722908680184,"stop":1722908986269,"duration":306085}},{"uid":"c08d2899a760a3b2","reportUrl":"../2042//#testresult/c08d2899a760a3b2","status":"passed","time":{"start":1722822015136,"stop":1722822290323,"duration":275187}},{"uid":"f0779774ed84e00","reportUrl":"../2041//#testresult/f0779774ed84e00","status":"passed","time":{"start":1722735466723,"stop":1722735786743,"duration":320020}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","abort_on_fail","group(1)","@pytest.mark.skipif(False, reason='skipping deploy, model expected to be provided.')"]},"source":"d9c2b9eaa9e1af6b.json","parameterValues":[]}